python/nav/alertengine/base.py

```diff
@@ -33,6 +33,15 @@
 
 _logger = logging.getLogger(__name__)
 
+# Queued notification dispatch decisions
+DISPATCHED_NOW = object()
+DISPATCHED_DAILY = object()
+DISPATCHED_WEEKLY = object()
+DISPATCH_FAILED = object()
+DISPATCH_IGNORED = object()
```

Comment:
That's certainly also a working way to make an enum of sorts. Have you considered using an actual [enum](https://docs.python.org/3/library/enum.html)? Namespaces are good, after all.

Reply:
Oki so the first commit mentions that this must run on Python 2.7. Enums can be made a requirement then, thanks to the `enum34`-package. The different commit-messages aren't conveniently available in this review-mode though, so maybe also mention it in the PR message.

Hopefully we can be completely over on Python 3 **soon**, NAV 5.1 cannot be out soon enough =)

Reply:
Had this been for Python 3 only, I would've definitely considered using an enum, and I'm always hesitant to introduce new dependencies as part of a bugfix on a stable branch. We'll just have to make that mental note (or create an issue) for the master branch.

(and any actual numbers don't matter here, anyway, just the semantic value, which is why I chose `object` for the job)